src/Layout/index.tsx

```diff
@@ -127,6 +125,19 @@ function Sidebar() {
     return (
         <div className={s.sidebar}>
             <nav className={s.sidebarInner}>
+                {launched && (
+                    <SidebarNav
+                        links={[
+                            {
+                                to: TERMS,
+                                label: t('common.entities.term.value_plural'),
+                            },
+                        ]}
+                    />
+                )}
+                <Suspense fallback={null}>
+                    <SidebarTerms />
```

Comment:
Ist it intentional that the SidebarTerms also show in the non-launched state?

Reply:
no...